<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReHireWindow</class>
 <widget class="QDialog" name="ReHireWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="firedList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="employeesList">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rehireButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Hire</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compareWorkers">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Compare</string>
   </property>
  </widget>
  <widget class="QPushButton" name="workerDetailsButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Details</string>
   </property>
  </widget>
  <widget class="QListWidget" name="compareWindow">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
